src/redfishpower/redfishpower.c

```diff
@@ -675,10 +675,11 @@ static void setpowerpath(char **av, char **path, char **postdata)
         xfree(*postdata);
         *postdata = NULL;
     }
-    if (av[0])
+    if (av[0]) {
         (*path) = xstrdup(av[0]);
-    if (av[1])
-        (*postdata) = xstrdup(av[1]);
+        if (av[1])
+            (*postdata) = xstrdup(av[1]);
+    }
```

Comment:
unrelated but the parens on the LHS are unnecessary.

Reply:
ya know, I always paren these de-reference assignments ... I don't know why.  Somehow makes it more clear to me that's an assignment?  Or maybe I picked it up via another project.

Reply:
OK, if they are helping!